script/foundry/utils/BroadcastManager.s.sol

```diff
@@ -22,8 +22,7 @@ contract BroadcastManager is Script {
             multisig = vm.envAddress("SEPOLIA_MULTISIG_ADDRESS");
             vm.startBroadcast(deployerPrivateKey);
         } else if (block.chainid == 31337) {
-            multisig = address(0x456);
-            deployer = address(0x999);
+            require(deployer != address(0), "Deployer not set");
```

Comment:
Why delete multisig setting? It should be the admin function. This script should still be usable in a local chain for testing.

Reply:
Okay, didn't think of using multisig in a local setting. Will add it back, thanks.

Reply:
It doesn't have to be a multisig per se, we can rename that variable `governanceAdmin` instead of multisig, so it's more explicit.
The point is to not change the assignment of that address as protocol admin, it can't be 0x0

Reply:
fb49eec

Also, I've noticed that we use `vm.startBroadcast(deployerPrivateKey)` regardless of using a multisig or not. Is this an intended behavior?

Reply:
The plan is to deploy with a non priviledged address (deployer), but set the multisig as admin.
So that would be by design, yes.